carsus/io/nist/ionization.py

```diff
@@ -290,3 +291,35 @@ def ingest(self, ionization_energies=True, ground_levels=True):
         if ground_levels:
             self.ingest_ground_levels()
             self.session.flush()
+
+
+class NISTIonizationEnergies(BaseParser):
+    """
+    Attributes
+    ----------
+    base : pandas.Series
+
+    Methods
+    -------
+    to_hdf(fname)
```

Comment:
colon missing?

Reply:
I'm using the same docstring format used in 'old' classes such as `NISTIonizationEnergiesIngester`. 

In the *numpydoc* guide the methods section says: `Document these as you would any other function`, so I assume this is not the appropiate format. 

I'll fix this.

Reply:
Forgot that. This is an example provided in the *numpydocs* guide:

```
class Photo(ndarray):
    """
    Array with associated photographic information.

    ...

    Attributes
    ----------
    exposure : float
        Exposure in seconds.

    Methods
    -------
    colorspace(c='rgb')
        Represent the photo in the given colorspace.
    gamma(n=1.0)
        Change the photo's gamma exposure.

    """
```

Reply:
Ok, done!